airflow/www/security.py

```diff
@@ -639,6 +750,16 @@ def sync_perm_for_dag(self, dag_id, access_control=None):
         if access_control:
             self._sync_dag_view_permissions(dag_resource_name, access_control)
 
+    def get_resource_permissions(self, resource):
```

Comment:
Just double-checking - resource here is different than "resource_name" (looks like - from the type).

Reply:
Yeah, `resource` should apply to the actual object